docs/.github/workflows/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
```

Comment:
- [ ] **[OPTIONAL]** Let's ensure we set up linters properly and ensure that the `.github/works/linters.yml` is in the root directory and not inside the `docs` directory so that the Github action will show up with the linters in the PR.